cmd/buildkitd/config/config.go

```diff
@@ -142,6 +142,8 @@ type ContainerdConfig struct {
 
 	MaxParallelism int `toml:"max-parallelism"`
 
+	CgroupParent string `toml:"cgroup-parent"`
```

Comment:
This is called `DefaultCgroupParent string `toml:"defaultCgroupParent"`` for OCI worker so we should keep the naming consistent

Reply:
Done. BTW, it looks like for OCI workers this could also be useful, but there is no way to set the `systemd-cgroup` flag on the `runc` command.